packages/visual-editor/src/utils/README.md

```diff
@@ -822,3 +822,51 @@ There are three type of MigrationActions:
   }
 }
 ```
+
+## withPropOverrides
+
+`withPropOverrides` lets you inject specific props into a component's `render` function. This is useful for customizing all instances of a component without making the value visible via fields in the Editor.
+
+### Example
+
+Given a component like this:
+
+```ts
+interface MockProps {
+  name: string;
+}
+
+const Mock: ComponentConfig<MockProps> = {
+  label: "Mock",
+  render: (props) => <>Hello {props.name}</>,
+};
+```
+
+And a mapping of which props are allowed to be overridden:
+
+```ts
+type ComponentPropOverrides = {
+  Mock: Pick<MockProps, "name">;
+};
+```
+
+You can inject `name` like this:
+
+```ts
+withPropOverrides(Mock, {
+  name: "World",
+});
+```
+
+### Supported usages for in-repo developers
```

Comment:
Can you add a blurb/example in the hybrid-development doc too? You can also link to this.